packages/arg-over-prop/package.json

```diff
@@ -17,5 +20,8 @@
     "@stamp/compose": "^1.0.2",
     "@stamp/core": "^1.0.1",
     "@stamp/is": "^1.0.0"
+  },
+  "devDependencies": {
+    "xo": "^0.37.1"
```

Comment:
I believe this is an accidental change. We have `xo` in the root package.json.

Reply:
Actually, this is a workaround. Without it, linter-xo in vscode does not activate within lerna packages for live linting.